spec.html

```diff
@@ -19147,8 +19153,7 @@ <h1>
       <emu-alg>
         1. Let _propertyNameReference_ be ? Evaluation of _expression_.
         1. Let _propertyNameValue_ be ? GetValue(_propertyNameReference_).
-        1. Let _propertyKey_ be ? ToPropertyKey(_propertyNameValue_).
-        1. Return the Reference Record { [[Base]]: _baseValue_, [[ReferencedName]]: _propertyKey_, [[Strict]]: _strict_, [[ThisValue]]: ~empty~ }.
+        1. Return the Reference Record { [[Base]]: _baseValue_, [[ReferencedName]]: _propertyNameValue_, [[Strict]]: _strict_, [[ThisValue]]: ~empty~ }.
```

Comment:
```suggestion
        1. NOTE: In most cases, ToPropertyKey(_propertyNameValue_) will be performed immediately after this step. However, in the case of `a[b] = c`, it will not be performed until after evaluation of `c`.
        1. Return the Reference Record { [[Base]]: _baseValue_, [[ReferencedName]]: _propertyNameValue_, [[Strict]]: _strict_, [[ThisValue]]: ~empty~ }.
```

Reply:
Wonderful, thanks!

Reply:
...though it seems like the linter thinks we're _performing_ `ToPropertyKey(_propertyNameValue_)` instead of mentioning it in a NOTE. 🤔